src/libs/seedData.ts

```diff
@@ -1,36 +1,35 @@
 import UserRepository from '../repositories/user/UserRepository'
-const userRepository= new UserRepository
-export default () => {
-  return new Promise((resolve, reject) => {
+import * as bcrypt from 'bcrypt'
+import config from '../config/configuration'
+const userRepository = new UserRepository
+export default async () => {
+  async function encodedPassword(Password) {
+    console.log("::::::::::", Password)
+    return await bcrypt.hash(Password, 10)
+  }
+  const ecncryptPassword = await encodedPassword(config.Password)
```

Comment:
encodedPAssword is a helper and used at mutliple place. Define it as a helper function instead of defining at every location.